catalog-info.yaml

```diff
@@ -151,7 +151,7 @@ spec:
         build_pull_request_forks: false
         build_tags: true
         # https://regex101.com/r/tY52jo/1
-        filter_condition: 'build.tag =~ "/^deploy@\d+\$/"'
+        filter_condition: 'build.tag =~ /^deploy@\d+\$/'
```

Comment:
Need to unescape the $ also

Reply:
Not sure how related, but there's another escaped $ in `.buildkite/pipelines/pipeline.kibana-serverless-release.yaml`

Reply:
It's required in pipeline.yml, but not here

Reply:
Ahhhh, thanks Brian.